app/models/XPath.scala

```diff
@@ -32,47 +32,46 @@ case class XPath(value: String) {
     }
 
   def task: Option[Task] = {
-    val pf: PartialFunction[String, Task] = _.replace("/CC015C/", "") match {
-      case x if x.matches("^TransitOperation/declarationType$")                                                            => PreTaskList
-      case x if x.matches("^TransitOperation/TIRCarnetNumber$")                                                            => PreTaskList
-      case x if x.matches("^TransitOperation/security$")                                                                   => PreTaskList
-      case x if x.matches("^TransitOperation/bindingItinerary$")                                                           => RouteDetails
-      case x if x.matches("^TransitOperation/reducedDatasetIndicator$")                                                    => TraderDetails
-      case x if x.matches("^TransitOperation/limitDate$")                                                                  => TransportDetails
-      case x if x.matches("^Authorisation(.+)$")                                                                           => TransportDetails
```

Comment:
and the .+ means one or more occurrence, so it wouldn't match on exactly "Authorisation" - looks good then